src/ipsis/course-merge/BaseCollectionEntity.js

```diff
@@ -47,13 +47,14 @@ export class BaseCollectionEntity extends Entity {
 	}
 
 	loadMorePageSize(collectionGetter) {
+		const pageSize = 20;
```

Comment:
Why not set the pageSize() property to have a default of 20. Assuming this is for when pageinfo does not have a page size

Reply:
This also breaks the comment just above the if statement. If there are only 3 more items to load, it would display "load 3 more items"

Reply:
Testing it again for these cases, Initially it was like this and I changed it to use entity.pageSize thinking the same in my last PR.

Reply:
I guess thats fine, I suppose this could revert back to using pageSize(), if we do that refactor to just get next 20 rather than {current} + 20. (idk the status of that)

Reply:
yes, that change would work when paging Api is updated to always fetch the next 20 records instead of doubling the pageSize on every click. I added personal notes to refactor this when that is done!